docs/en/01_basic_usage.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Basic usage
+summary: How to set up your taxonomy relations and fields
+---
+
+# Basic usage
+
+The main model you'll be interacting with is [`TaxonomyTerm`](api:SilverStripe\Taxonomy\TaxonomyTerm). This class represents the actual taxonomy terms you will apply to your data.
+
+A taxonomy is of extremely limited use by itself. To make use of it, you need to associate it with
+`DataObject` models in your site.
+
+To add the the ability to associate a model with `TaxonomyTerm`, you need to add the many-many relation:
+
+```php
+namespace App\Model;
+
+use SilverStripe\Forms\TreeMultiselectField;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Taxonomy\TaxonomyTerm;
+
+class MyModel extends DataObject
+{
+    // ...
+    private static $many_many = [
+        'Terms' => TaxonomyTerm::class
+    ];
+
+    public function getCMSFields()
+    {
+        $fields = parent::getCMSFields();
+        // ...
+        $fields->addFieldToTab('Root.Main', TreeMultiselectField::create(
```

Comment:
`TreeMultiselectField` is way more intuitive and lightweight than the old docs saying to use `GridField`.